downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -164,7 +164,36 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
         }
     }
     ```
+
+- A bug where the runtime error messages were including the member types of a union type instead of the original union type name has been fixed.
```

Comment:
How about something like "flattened representation of types instead of the name used in the type definition"?

Reply:
Isn't `flattened representation` an internal concept? We may have to explain what do we mean by `flattened representation` if we are using that

Reply:
Self-explanatory IMO. Or 

A bug that resulted in runtime error messages including the string representation of the member types of a union instead of the name used in the type definition has been fixed.

Reply:
Will go with the fist one 😃